server/views/applications/taskList.njk

```diff
@@ -9,10 +9,10 @@
 {% endblock %}
 
 {% block beforeContent%}
-  {% if referrer %}
+  {% if referer %}
     {{ govukBackLink({
         text: "Back",
-        href: referrer
+        href: referer | validateReferer
```

Comment:
do you think we should leave the validation of the `referer` to the controllers? In this case this `referer` has already been validated by `showMissingRequiredTasksOrTaskList`. I feel like if we are expecting to also validate it in the templates it could open door to double-validation like this. Also feels like the nunjucks should be able to trust anything they've been passed from the controller?

Reply:
ah yes, good point - we don't need it here. I think I was a bit confused about where that `referer` was coming from early on and didn't update it when I figured it out. Can remove this one now.

Reply:
And yeah, I think that's a good rule of thumb